.travis.yml

```diff
@@ -0,0 +1,56 @@
+language: go
+
+services:
+  - docker
+
+go:
+  - "1.9.x"
+
+before_install:
+  - . $HOME/.nvm/nvm.sh
+  - nvm install 8
+  - nvm use 8
+  - npm install -g yarn
+
+install:
+  - make dependencies
+
+stages:
+  - name: validate and build
+    if: >-
```

Comment:
Is this condition meant to be... always run? You could just omit the `if` field here.

Reply:
mmm... there are some cases where this stage won't run... but who cares... we can get rid of this tricky `if`, and run the tests always; I'll do so.